apps/src/templates/feedback/LevelFeedback.jsx

```diff
@@ -0,0 +1,98 @@
+import React, {Component} from 'react';
+import i18n from '@cdo/locale';
+import color from '@cdo/apps/util/color';
+import {levelFeedbackType} from './types';
+import LevelFeedbackEntry from '@cdo/apps/templates/feedback/LevelFeedbackEntry';
+import Button from '@cdo/apps/templates/Button';
+
+export default class LevelFeedback extends Component {
+  static propTypes = levelFeedbackType;
+
+  state = {
+    showingOlderComments: false
+  };
+
+  toggleshowOlderComments = () => {
+    this.setState({showingOlderComments: !this.state.showingOlderComments});
+  };
+
+  render() {
+    const {
+      lessonName,
+      lessonNum,
+      levelNum,
+      linkToLevel,
+      unitName,
+      feedbacks
+    } = this.props;
+
+    const {showingOlderComments} = this.state;
+
+    const displayedFeedbacks = showingOlderComments
+      ? feedbacks
+      : feedbacks.slice(0, 1);
+
+    const hasMultipleFeedbacks = feedbacks.length > 1;
+    const marginBottom = hasMultipleFeedbacks ? 8 : 24;
```

Comment:
Why do we need this?

Reply:
For levels that have just one feedback we do not display the "Show older comments" button, so the design has a little bit of extra spacing to make up for the space that we usually get with the button.